apps/www/src/content/cli.md

```diff
@@ -85,7 +86,8 @@ Options:
 
 ## update
 
-Use the `update` command to update components in your project. This will overwrite any modifications you've made to the components, so be sure to commit your changes before running this command.
+Use the `update` command to update components in your project. This will overwrite any modifications you've made to the
+components, so be sure to commit your changes before running this command.
```

Comment:
```suggestion
Use the `update` command to update components in your project. This will overwrite any modifications you've made to the components, so be sure to commit your changes before running this command.
```